NitroxServer/GameLogic/EscapePodManager.cs

```diff
@@ -1,22 +1,72 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using ProtoBufNet;
 using NitroxModel.DataStructures.GameLogic;
+using NitroxServer.GameLogic.Items;
 using UnityEngine;
+using System.Linq;
 
 namespace NitroxServer.GameLogic
 {
+    [ProtoContract]
```

Comment:
I think we'll want to put the serialized data into a new model.  This will allow us to decouple the persistence layer and not litter the Manager with serialization attributes.